emiluz/README.md

```diff
@@ -0,0 +1,148 @@
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+a) "1" == 1 true. One is equal to One.
+
+b) "1" === 1 false. String One is not strict equal to One.
+
+c) x == 'x' The X is not defined therefore X is not equal to string.
+
+d) x == (x+'') The X is not defined.
+
+e)'' == ' ' The answer is false, null
+
+f) x = true
+
+g) var x; x == 'undefined'` false. X is not equal to undefined
+
```

Comment:
In fact, x is equal to undefined. The subtle, but very important, difference in this situation is that we are not comparing x to the boolean undefined, but rather the string 'undefined'. x != 'undefined'